lib/index.ts

```diff
@@ -13,9 +16,24 @@ export async function getCallGraphMvn(
     getClassPathFromMvn(targetPath),
   );
 
-  return await timeIt('getCallGraph', () =>
-    getCallGraph(classPath, targetPath, timeout),
+  const tmpDir = await promisifedFs.mkdtemp(
+    path.join(tempDir, 'call-graph-generator'),
   );
+  const filePath = path.join(tmpDir, 'callgraph-classpath');
+  await promisifedFs.writeFile(filePath, classPath);
+
+  try {
+    return await timeIt('getCallGraph', () =>
+      getCallGraph(classPath, targetPath, timeout),
```

Comment:
where is `filePath` being used ?

Reply:
In `unlink`. File needs to be deleted before the directory.

Reply:
I see you create and delete the file, but where do you pass `filePath` to the call graph generator?

Reply:
This specific part wasn’t touch - it is still just a string param handed over to `getCallGraph`. It will pass it to `getCallGraphGenCommandArgs`which will wrap it accordingly.

Reply:
discussed in person and fixed :-)

Reply:
Not pushed yet right @miiila